cinema/models.py

```diff
@@ -5,6 +5,32 @@
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", related_name="movies")
+    actors = models.ManyToManyField("Actor", related_name="actors")
 
     def __str__(self):
-        return self.title
+        return f"{self.title} {self.actors.count()} actors"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return {self.name}
```

Comment:
The `__str__` method in the `Genre` class is using curly braces `{}` instead of parentheses `()`. This will cause a syntax error. It should be `return self.name`.